nautobot/docs/installation/4-gunicorn.md

```diff
@@ -1,20 +1,46 @@
 # Gunicorn
 
-Like most Django applications, Nautobot runs as a [WSGI application](https://en.wikipedia.org/wiki/Web_Server_Gateway_Interface) behind an HTTP server. This documentation shows how to install and configure [gunicorn](http://gunicorn.org/) (which is automatically installed with Nautobot) for this role, however other WSGI servers are available and should work similarly well. [uWSGI](https://uwsgi-docs.readthedocs.io/en/latest/) is a popular alternative.
+!!! warning
+    As of Nautobot v1.0.0b1 these instructions are in a pre-release state and will be evolving rapidly!
+
+Like most Django applications, Nautobot runs as a [WSGI
+application](https://en.wikipedia.org/wiki/Web_Server_Gateway_Interface) behind an HTTP server. This documentation shows
+how to install and configure [gunicorn](http://gunicorn.org/) (which is automatically installed with Nautobot) for this
+role, however other WSGI servers are available and should work similarly well.
+[uWSGI](https://uwsgi-docs.readthedocs.io/en/latest/) is a popular alternative.
 
 ## Configuration
 
-Nautobot ships with a default configuration file for gunicorn. To use it, copy `/opt/nautobot/contrib/gunicorn.py` to `/opt/nautobot/gunicorn.py`. (We make a copy of this file rather than pointing to it directly to ensure that any local changes to it do not get overwritten by a future upgrade.)
```

Comment:
Are we going to delete the `contrib/gunicorn.py` file as no longer used?

Reply:
It was already deleted. ;)